scheds/c/scx_central.bpf.c

```diff
@@ -57,13 +58,15 @@ enum {
 
 const volatile s32 central_cpu;
 const volatile u32 nr_cpu_ids = 1;	/* !0 for veristat, set during init */
-const volatile u64 slice_ns = SCX_SLICE_DFL;
+const volatile u64 slice_ns;
 
 bool timer_pinned = true;
 u64 nr_total, nr_locals, nr_queued, nr_lost_pids;
 u64 nr_timers, nr_dispatches, nr_mismatches, nr_retries;
 u64 nr_overflows;
 
+SCX_ENUM_DEFINE();
```

Comment:
`SCX_OPS_LOAD()` already assumes `UEI_DEFINE()`, so adding another dependency is just fine. We can add e.g. `SCX_COMMON_DEFINE()` to make it easier.